### ISSUE TITLE
Release Notes for CCE Cluster Versions not up-to-date

### ISSUE THREAD

**Issue by david-slr** [opentelekomcloud-docs/cloud-container-engine#87]
Issue on page:        https://github.com/opentelekomcloud-docs/cloud-container-engine/blob/main/umn/source/clusters/cluster_overview/release_notes_for_cce_cluster_versions.rst

The CCE Versions are not up-to-date. 
For example v1.27.5-r0 is missing an v1.28.xx

**Comment by oeotvos** [comment]
You are correct, we need to update the table with all available versions. Thank you for notification.

**Comment by oeotvos** [comment]
Will be updated soon together with new v1.29 release.

**Comment by oeotvos** [comment]
Missing versions were added, new versions will be also included with upcoming v1.30 release.
https://docs.otc.t-systems.com/cloud-container-engine/umn/clusters/cluster_overview/patch_version_release_notes.html#cce-10-0405
